packages/expo-cli/src/commands/export.ts

```diff
@@ -13,6 +13,7 @@ export default function (program: Command) {
       .command('export [path]')
       .description('Export the static files of the app for hosting it on a web server')
       .helpGroup('core')
+      .option('--platform <all|android|ios>', 'Platforms: ios, android, all. Default: all')
```

Comment:
Over time we should write things in the order of: Android, iOS, web 
```suggestion
      .option('--platform <all|android|ios>', 'Platforms: android, ios, all. Default: all')
```